knapsack/knapsack.py

```diff
@@ -20,6 +20,10 @@ def knapsack(capacity: int, weights: list[int], values: list[int], counter: int)
     which is the limit of the capacity.
     """
 
+    # Invalid Case
+    if len(val)!=len(w):
```

Comment:
please recheck the variable name.

Reply:
How do i do that now. Can you please suggest me. Should i create new PR or what. Really new here

Reply:
as you can see if knapsack called like knapsack(cap, w, val, c) then knapsack function accept the variables to named capacity, weights, values and counter.
so, there is no val and w now but only use values and weights.
then now you can code like this -> if len(values) != len(weights)